### ISSUE TITLE
Working proxy list

### ISSUE THREAD

**Issue by apexc** [shadow-1/yi-hack-v3#230]
Hi, 

Anyone here still got their unit working?
it has been 6 months now where my cams are not working. 

could not get any proxy to work.

**Comment by tingsin86** [comment]
yes. go this link. you can get all the CN proxy http://spys.one/free-proxy-list/CN/
